bin/zopen-install

```diff
@@ -21,6 +21,65 @@ setupMyself()
 setupMyself
 checkWritable
 
+verifySignatureOfPax()
+{
+  # Extracting values and checking for errors
+  FILE_TO_VERIFY=$(jq -r '.product.pax' "${metadataFile}")
+  if [ $? -ne 0 ] || [ "${FILE_TO_VERIFY}" = "null" ] || [ -z "${FILE_TO_VERIFY}" ]; then
+     printError "Failed to extract 'pax' from ${metadataFile}" >&2
+     return 1
+  fi
+
+  SIGNATURE=$(jq -r '.product.signature' "${metadataFile}")
+  if [ $? -ne 0 ] || [ "${SIGNATURE}" = "null" ] || [ -z "${SIGNATURE}" ]; then
+     printfError "Failed to extract 'signature' from ${metadataFile}" >&2
+     return 1
+  fi
+
+  PUBLIC_KEY=$(jq -r '.product.public_key' "${metadataFile}")
+  if [ $? -ne 0 ] || [ "${PUBLIC_KEY}" = "null" ] || [ -z "${PUBLIC_KEY}" ]; then
+     printError "Failed to extract 'public_key' from ${metadataFile}" >&2
+     return 1
+  fi
+
+
+  # Create a temporary directory for GPG keyring
+  TMP_GPG_DIR="$HOME/tmp_gpg_verify"
```

Comment:
Might be better to reuse $tmp from here instead of $HOME: https://github.com/ZOSOpenTools/meta/blob/main/bin/zopen-build#L273